File: test/fixtures/clojure/get_with_data.clj
(require '[clj-http.client :as client])

(client/get "http://localhost:28139/synthetics/api/v3/monitors" {:query-params {:test "2"
                                                                                :limit "100"
                                                                                :w "4"}
                                                                 :headers {"X-Api-Key" "123456789"}})

File: test/fixtures/r/get_with_data.r
library(httr)

headers = c(
  `X-Api-Key` = "123456789"
)

params = list(
  test = "2",
  limit = "100",
  w = "4"
)

res <- httr::GET(url = "http://localhost:28139/synthetics/api/v3/monitors", httr::add_headers(.headers=headers), query = params)
